docs/blog/20230201-Daeraxa-FebUpdate.md

```diff
@@ -0,0 +1,115 @@
+---
+title: February Update
+author: Daeraxa
+date: 2023-02-01
+category:
+  - news
+  - log
+tag:
+  - update
+---
+
+What has the Pulsar team and community been up to lately?
+
+<!-- more -->
+
+# Community Update
+
+Hi everyone, welcome to our first post in what we hope will be a regular (or at
+least semi-regular) set of updates from the Pulsar team to let you know what
+is going on in the background. We see updates, work and improvements to the
+entire Pulsar ecosystem nearly every single day but many of these either aren't
+very obvious or requires a very close following of the various communication
+channels.  
+This update format is designed to let people know of our major wins and exciting
+updates of things we are working on so you can share in the progress we are
+making.  
+We hope you enjoy reading this and getting to know what is going on with the
+project and as ever we will see you among the stars!
+
+## Tree-sitter
+
+Those who have been keeping up with the development of Pulsar know that one of
+our major goals is to move upgrade to newer versions of Electron. Unfortunately
```

Comment:
```suggestion
our major goals is to upgrade to newer versions of Electron. Unfortunately
```

Reply:
Whoops, stream of consciousness made it in. I'm awful at reviewing my own stuff

Reply:
Hey still less issues then when I write something out, so a win by my standards aha